courseware/data_literacy/about_the_course.md

```diff
@@ -0,0 +1,68 @@
+---
+layout: default
+title: About the Course
+nav_order: 1
+parent: Data Literacy with Data Commons
+grand_parent: Courseware
+---
+
+# About the Course
+
+<div markdown="span" class="alert alert-info" role="alert" style="text-align: center">
+    >> [Access the course materials here](course_materials.html). <<
+</div>
+
+## What is it?
+
+A set of [modules](course_materials.html#modules) focusing on several [key concepts](topics.html#key-concepts) focusing on data modeling, analysis, visualization and the (ab)use of data to tell (false) narratives. Each module lists its objectives and builds on a pedagogical narrative around the explanation of key concepts, e.g. the differences between correlations and causation. We extensively use the Data Commons platform to point to _real world_ examples without needing to write a single line of code!  
+
+All material is provided publicly and free of charge, under a Creative Commons license ([CC BY](https://creativecommons.org/licenses/by/4.0/)).
+
+## Who is this for?
+
+Anyone and everyone. Instructors, students, aspiring data scientists and anyone interested in advancing their data comprehension and analysis skills without needing to code. For instructors, the rest of this page details the curriculum organization and how to find key concepts/ideas to use.
+
+## What's Different?
+
+There are several excellent courses which range from basic data analysis to advanced data science. We make no claim about "Data Literacy with Data Commons" being a replacement for them. Instead, we hope for this curriculum to become a useful starting point for those who want to whet their appetite in becoming data literate. This material uses a hands on approach, replete with _real world_ examples but without requiring any programming. It also assumes only a high-school level of comfort with Math and Statistics. Data Commons is a natural companion platform to enable easy access to data and core visualizations. We hope that anyone exploring the suggested examples will rapidly be able to explore more and even generate new examples and case studies on their own! If you end up finding and exploring new examples and case studies, please do share them with us at [courses@datacommons.org](mailto:courses@datacommons.org).
```

Comment:
whet -> wet?

Reply:
I believe the correct phase is actually "whet your appetite"